### ISSUE TITLE
FPP Crash on Plugins.cpp (related to remote falcon)

### ISSUE THREAD

**Issue by Jchuchla** [FalconChristmas/fpp#1709]
FPP 7.3.3 on Pi 3b+

Not sure if this is an FPP or Remote Falcon Plugin issue.  Reporting on both projects.  The corresponding RF ticket is at: https://github.com/whitesoup12/remote-falcon/issues/137

Describe the bug:
I just today started using the Remote Falcon "Managed PSA" functionality. I run a semi-static sequence all night. This is a simple 1 minute single sequence playlist that loops over and over all night. I didn't want the PSA sequence to come on every 3 minutes so I added the "Managed PSA On" and off commands to my playlist(s). On the next schedule change (at midnight), FPPD crashed. The following is logged in the fppd.log.

2023-12-11 00:01:01.447 (29497) [Plugin] /opt/fpp/src/Plugins.cpp:281: We failed to exec our command callback: Permission denied
2023-12-11 00:01:01.448 (29497) [Plugin] /opt/fpp/src/Plugins.cpp:283: /home/fpp/media/plugins/remote-falcon/commands//managed_psa_off.php
2023-12-11 00:01:01.448 (29497) [Plugin] /opt/fpp/src/Plugins.cpp:283: (null)
2023-12-11 00:12:38.671 (1911) [General] /opt/fpp/src/fppd.cpp:265: Crash handler called: 7
2023-12-11 00:12:38.671 (29497) [General] /opt/fpp/src/fppd.cpp:265: Crash handler called: 7

**  Recovery steps **
Because this playlist was the one that was supposed to be playing at the current moment, it was tricky to recover. After a reboot it would go right into the playlist with the problem command.  I had no control from FPPs web gui. I first had to use the command line to hand edit the json in the playlist file to delete the Managed PSA Off command.
Then I rebooted the pi and the playlist started as it should.

**Comment by Jchuchla** [comment]
I should also add that Managed PSA should have already been off when it called the command to turn it off.  Though I'm not sure how to verify what state it was in when the crash occurred.  I'm not sure if issuing the off command when it was already off was part of the problem.

**Comment by Jchuchla** [comment]
Had one of those laying in bed aha moments.  I found the issue to be with incorrect permissions on two scripts in plugin folder.
"managed_psa_off.php" and "managed_psa_on.php" were set to 644 permissions.  I noticed the rest of the scripts in that folder that corresponded with RF commands had permissions of 755.  I changed the permissions on these two scripts to 755 and they started working from the command preset page.  I will test the command functionality in the playlists tonight during showtime.
